### ISSUE TITLE
How to use the LR images in training?

### ISSUE THREAD

**Issue by zhenxili96** [IceClear/StableSR#10]
Hi, I'm trying to understand the pipeline of your great work. But I'm not sure how to use the training data. Is there any diffusion process (adding noise to traning images) in your training process?

From what I understood, the training process is: 
first, construct LR images by degrading HR images (eg. DIV8K),
then, in the first training stage, input these LR images (without adding noise) to the VQGAN, Unet, to get the output images, which are compared with GT (HR) images to calculate the loss,
and, in the second training stage, input these LR images (without adding noise) , latent, samples, GT to train CFW.

I didn't find the diffusion process (adding noise to traning images) in the training process. Is there any mistakes in my understanding?

**Comment by IceClear** [comment]
Hi.
For the second stage, you need to regenerate LR patches rather than using the same training data in the first stage.
The degrading process is the same with the first one, but you need to save the data locally.
In the first stage, the degrading process is on-the-fly.

**Comment by GongruLi** [comment]
> 您好，我正在尝试理解您出色工作的流程。但我不确定如何使用训练数据。您的训练过程中是否存在扩散过程（即在训练图像中添加噪声）？
> 
> 我的理解是，训练过程是： 首先通过对HR图像进行降质处理（例如DIV8K）构建LR图像， 然后在第一个训练阶段将这些LR图像（不加噪声）输入到VQGAN、Unet中得到输出图像，并与GT（HR）图像进行比较计算损失， 在第二个训练阶段，将这些LR图像（不加噪声）、latent、samples、GT输入到CFW中进行训练。
> 
> 我在训练过程中没有发现扩散过程（给训练图像添加噪声）。我的理解有误吗？

你好，我想问一下第一阶段的训练是有监督训练还是无监督训练，第一阶段训练是将LQ和HQ图像对比计算损失，还是只是用LQ进行无监督训练

**Comment by GongruLi** [comment]
> 您好。 对于第二阶段，您需要重新生成 LR 图像块，而不是使用第一阶段的相同训练数据。 降级过程与第一阶段相同，但您需要将数据保存在本地。 在第一阶段，降级过程是即时进行的。

Hello, I'd like to ask whether the first stage of training is supervised or unsupervised. Does the first stage of training involve calculating the loss by comparing LQ and HQ images, or is it just unsupervised training using LQ images?
